clipctl

```diff
@@ -1,25 +1,53 @@
 #!/usr/bin/env bash
 
+: "${CM_DIR:="${XDG_RUNTIME_DIR-"${TMPDIR-/tmp}"}"}"
+
 if [[ -z $1 ]] || [[ $1 == --help ]] || [[ $1 == -h ]]; then
     cat << 'EOF'
 clipctl provides controls for the clipmenud daemon.
 
 You can temporarily disable clip collection without stopping clipmenud entirely
```

Comment:
nit: Maybe remove this block since you now enumerate them all anyway.

Reply:
Done